scripts/scale.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+export VALID_NAMESPACES=("hocs-gamma" "hocs-delta" "hocs-qax")
+
+current=$(kubectl config current-context)
+if [[ ${VALID_NAMESPACES[*]} =~ "$current" ]] ; then
+  echo "Unauthorised Namespace" >&2
+  exit 1
+fi
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: scale.sh down/up" >&2
+  exit 1
+fi
+if [ "$1" != "down" -a "$1" != "up" ]; then
+  echo "Usage: scale.sh down/up" >&2
+  exit 1
+fi
+
+if [ "$1" = "down" ]; then
+  kubectl scale --replicas=0 deployment/hocs-management-ui
```

Comment:
Instead of specifying every deployment manually, which'll have to be maintained, perhaps we could scale based on label (e.g. `hocs-backend`) instead?

If we're scaling everything to 0 do we need to `wait` at all?

Reply:
Yes I think so. The communication channel to the dependent service needs to be open when the independent services goes down.